### DISCUSSION TITLE
the the notracking project is shutting down, should we use other blocklist source ?

### DISCUSSION THREAD

**Discussion by confever** [RaspAP/raspap-webgui#1407] [Q&A] [answered]
Hello, i'm still new to this raspberry and raspap. 
I install raspap on my 4b, it works great.
I enable adblock and see that it use notracking hosts blocklist. but when i check on their github page, it said that the project is shutting down. and the github is already archived as read only.

does this mean we need to find other blocklist sources ? if so, any recomendation on which one to use ?

thanks in advance.

**Comment by billz** [comment, marked as answer]
Thanks for raising this—it's on the near-term todo list. I'll create an issue to track it.
